src/server/procedures/myForestV2/contributions.ts

```diff
@@ -437,11 +472,14 @@ export const contributionsProcedure = procedure
       }
     });
 
+    const sortedContributionsMap =
+      getSortedContributionsMap(myContributionsMap);
+
     console.log(new Date().toLocaleString(), 'ending contributionsProcedure');
 
```

Comment:
remove the console log

Reply:
This is left for logging purposes on the server, to identify if there are performance issues that need to be resolved.